build.gradle

```diff
@@ -125,6 +125,26 @@ subprojects {
       oldName = project.name
       oldVersion = "1.1.0"
     }
+
+    tasks.register('showDeprecationRulesOnRevApiFailure') {
+      doLast {
+        throw new RuntimeException("==================================================================================" +
```

Comment:
nit: Should we use platform independent carriage return like `%n` maybe instead of '\n'

Reply:
Doesn't %n only work in String.Format?

Reply:
yes. I didn't explicitly mention it thinking it was obvious.

Reply:
This is okay since we're primarily using it in the CI environment right?

Reply:
Not necessarily. Local compile `gradle clean build -x intTest -x test` also executes revAPI.